envision/web/src/client.js

```diff
@@ -100,7 +99,7 @@ export default class Client {
               : value
           );
           stateQueue.push({
-            state: state,
+            state: data.state,
             current_elapsed_time: data.current_elapsed_time,
```

Comment:
I think `current_elapsed_time` is an inaccurate name for this.  It's appears to rather be the `current_frame_time` (which can probably just be `frame_time`).

Reply:
(of course, how exactly we want to name this will depend on its ultimate semantics, which depends on the considerations below...)